Libraries/Utilities/StringExtensions.cs

```diff
@@ -41,6 +41,16 @@ public static class StringExtensions
         public static string JoinNonEmpty(this IEnumerable<string> values, string separator = "") =>
             values?.Where(s => !string.IsNullOrEmpty(s)).Join(separator);
 
+        /// <summary>
+        /// Join strings fluently.
+        /// </summary>
+        /// <param name="values">The <see cref="string"/> values to join.</param>
+        /// <param name="separator">The separator to use between the <paramref name="values"/>.</param>
+        /// <returns>A new <see cref="string"/> that concatenates all values with the <paramref name="separator"/>
+        /// provided.</returns>

```

Comment:
This should be formatted as follows (like we do in HTML):
```suggestion
        /// <returns>
        /// A new <see cref="string"/> that concatenates all values with the <paramref name="separator"/> provided.
        ///</returns>
```
Please fix the formatting for the methods above, too.

Reply:
Oh! I didn't mean for you to reformat **all** docs elements, only those `<returns>` elements with too long texts to fit into one line.

Please revert the places that fit into a single line including opening and closing tags.